<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InformerMessageEditorDialog</class>
 <widget class="QDialog" name="InformerMessageEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="11" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblGroups">
     <property name="text">
      <string>Адресаты</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblText">
     <property name="text">
      <string>Сообщение</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblSubject">
     <property name="text">
      <string>&amp;Тема</string>
     </property>
     <property name="buddy">
      <cstring>edtSubject</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QWidget" name="wrapMessage" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wrapMessageToolbox" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnAlignLeft">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Выровнять по левому краю</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../s11main.qrc">
             <normaloff>:/new/prefix1/icons/text_editor/align_left.png</normaloff>:/new/prefix1/icons/text_editor/align_left.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlign</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAlignCenter">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Выровнять по центру</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../s11main.qrc">
             <normaloff>:/new/prefix1/icons/text_editor/align_center.png</normaloff>:/new/prefix1/icons/text_editor/align_center.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlign</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAlignRight">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Выровнять по правому краю</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../s11main.qrc">
             <normaloff>:/new/prefix1/icons/text_editor/align_right.png</normaloff>:/new/prefix1/icons/text_editor/align_right.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlign</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAlignJustify">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Растянуть текст</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../s11main.qrc">
             <normaloff>:/new/prefix1/icons/text_editor/align_justify.png</normaloff>:/new/prefix1/icons/text_editor/align_justify.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">btnGrpAlign</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnBold">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="toolTip">
            <string>Полужирный</string>
           </property>
           <property name="text">
            <string>B</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnItalic">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="toolTip">
            <string>Курсив</string>
           </property>
           <property name="text">
            <string>I</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnUnderstroke">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <underline>true</underline>
            </font>
           </property>
           <property name="toolTip">
            <string>Подчеркнутый</string>
           </property>
           <property name="text">
            <string>U</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFontSize">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>46</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Размер:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinFontSize">
           <property name="suffix">
            <string>pt</string>
           </property>
           <property name="minimum">
            <number>8</number>
           </property>
           <property name="maximum">
            <number>48</number>
           </property>
           <property name="value">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblFont">
           <property name="text">
            <string>Шрифт:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFontComboBox" name="cbxFont">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnLink">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Вставить ссылку</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../s11main.qrc">
             <normaloff>:/new/prefix1/icons/text_editor/link.png</normaloff>:/new/prefix1/icons/text_editor/link.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>19</width>
             <height>19</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="edtText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="edtSubject"/>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="cbxUserGroup">
     <item>
      <property name="text">
       <string>&lt;все&gt;</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../s11main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InformerMessageEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InformerMessageEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="btnGrpAlign"/>
 </buttongroups>
</ui>
